gem/locales/dz.yml

```diff
@@ -0,0 +1,21 @@
+# :one_other pluralization (see https://github.com/ddnexus/pagy/blob/master/gem/lib/pagy/i18n.rb)
```

Comment:
Inconsistent with the `:other` pluralization indicated in the `gem/lib/pagy/i18n.rb` entry

Reply:
thank you, will update